### PR TITLE
AMP-112976 [streaming] Updated streaming transformations doc for GA

### PR DESCRIPTION
**Pull request by kg-datamonster**
Event Streaming Transformations is now GA. Updating the doc to reflect the same.

### CODE REVIEW

File: content/collections/data/en/streaming-transformations.md [hunk 1]
```diff
@@ -33,24 +33,20 @@ This feature is available on Plus, Growth, or Enterprise plans. For more informa
...
 
-* You must enable this integration in each Amplitude project you want to use it in.
 * Amplitude sends selected event and user properties along with the event.
 * Amplitude targets an end-to-end p95 latency of 60s. This means 95% of Events streamed from Amplitude within 60s or less. Amplitude has internal processes, monitors, and alerts in place to meet this target.
 
 ## Limitations
 
 There are some limitations when streaming transformations from Amplitude:
 
+* Like saved charts, if there is a change in the name of Custom Event or Derived Property, existing syncs that use these will not work as expected because you need to update the sync configs to use the new names. If the underlying definition of the Custom Event or Derived Property change, you do not need to update the sync. Please make sure to check if these transformations are currently used in any syncs before updating their names.
```
Comment (SpencerFleury) [inline content/collections/data/en/streaming-transformations.md:43]:
```suggestion
* When changing the names of custom events or derived properties, don't forget to update any existing sync configs that use them. Syncs require up-to-date event and property names in order to work properly. Note that this only applies to the name of the transformation: changing the underlying definition of a custom event or derived property doesn't affect syncing.
```

File: content/collections/data/en/streaming-transformations.md [hunk 2]
```diff
@@ -33,24 +33,20 @@ This feature is available on Plus, Growth, or Enterprise plans. For more informa
...
-* You can't stream **channel classifiers** directly. Channels, which act like derived properties applied in real-time when querying within Amplitude, are mainly used by marketers to define their acquisition channels based on UTM and referrer data. Although they're used as a property where values map to specific rules on existing properties, these channel properties aren't selectable when setting up the sync in either event filters or when sending more properties to the destination.
+* You can't stream **channel classifiers** directly. However, this is available on request. Channels, which act like derived properties applied in real-time when querying within Amplitude, are mainly used by marketers to define their acquisition channels based on UTM and referrer data. Although they're used as a property where values map to specific rules on existing properties, these channel properties aren't selectable when setting up the sync in either event filters or when sending more properties to the destination without being explicitly enabled.
```
Comment (SpencerFleury) [inline content/collections/data/en/streaming-transformations.md:45]:
```suggestion
* You can only stream **channel classifiers** directly by requesting it from Amplitude. Channels, which act like derived properties applied in real-time when querying within Amplitude, are mainly used by marketers to define their acquisition channels based on UTM and referrer data. Although they're used as a property where values map to specific rules on existing properties, these channel properties can't be selected when setting up the sync in either event filters, or when sending more properties to the destination, unless you've explicitly enabled this feature.
```

File: content/collections/data/en/streaming-transformations.md [hunk 3]
```diff
@@ -65,3 +61,8 @@ You can use both for your streaming sync. For example, if three event types are
 {{partial:collapse name="How are custom events and transformed properties handled during streaming?"}}
 Custom events and transformed properties follow the configurations set in your Amplitude Data taxonomy. The transformations are applied before the data is streamed to the destination.
 {{/partial:collapse}}
+
+
+{{partial:collapse name="How can I enable channel classifiers for my event stream?"}}
+Support for channel classifiers to be selected in the event streaming sync config is available on request. Email integrations@Amplitude.com with your Organization ID and App IDs if you're interested in understanding how to get access to this capability.
```
Comment (SpencerFleury) [inline content/collections/data/en/streaming-transformations.md:67]:
```suggestion
Support for selecting channel classifiers in the event streaming sync config is available on request. Email integrations@Amplitude.com with your organization ID and app IDs for access.
```

**Review by SpencerFleury — CHANGES_REQUESTED** [review]

**Comment by kg-datamonster** [comment]
Thanks for the review and suggestions @SpencerFleury 🙌

**Review by SpencerFleury — APPROVED** [review]

**Pull request merged**